app/controllers/components/course/materials_component.rb

```diff
@@ -0,0 +1,15 @@
+class Course::MaterialsComponent < SimpleDelegator
+  include Course::ControllerComponentHost::Component
+
+  def sidebar_items
+    [
+      {
+        key: :materials,
+        title: I18n.t('course.material.sidebar_title'),
+        weight: 4,
+        path: course_material_folder_path(current_course, current_course.root_folder),
```

Comment:
But if you directly update at db level, manually manipulating all those fields (creator, updater, updated_at, created_at, etc), it still very possible that you will break things.